cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraData.java

```diff
@@ -78,13 +78,12 @@ public static final CassandraData createCleanupStatusData() {
         return new CassandraData(CassandraTask.TYPE.CLEANUP);
     }
 
-
-    public static final CassandraData createBackupSnapshotData(
+    public static final CassandraData createBackupSchemaData(
         final String hostname,
         final BackupRestoreContext context) {
 
         return new CassandraData(
-            CassandraTask.TYPE.BACKUP_SNAPSHOT,
+            CassandraTask.TYPE.BACKUP_SCHEMA,
```

Comment:
I copy pasted then updated to schema, so git diff produced this output. See the file, both snapshot and schema sections are there.